cloudapi/errors.go

```diff
@@ -13,8 +13,8 @@ var (
 	errUnknown          = errors.New("an error occurred communicating with k6 Cloud")
 )
 
-// ErrorResponse represents an error cause by talking to the API
-type ErrorResponse struct {
+// ResponseError represents an error cause by talking to the API
+type ResponseError struct {
```

Comment:
@yorugac If I'm not wrong, the operator relies on this pkg. Does this change break anything? And if it does, is it easy to fix?

Reply:
It doesn't look as it is used https://github.com/search?q=repo%3Agrafana%2Fk6-operator%20ErrorResponse&type=code

And neither is it in anything else by the looks of it.

I am okay with reverthing this if we want? I can just put in a nolint